src/web-interface/src/libcluon-viewer-0.01.js

```diff
@@ -335,6 +454,7 @@ function addFieldCharts(sourceKey, data) {
         }
       };
       
+      console.log("source %s %s", sourceKey, i)
```

Comment:
console.log() - for debugging or useful?

Reply:
Can still be useful for future debugging, after merge